File: ruby/elem4.rb
puts "Digite um número:"
# Encadeamos a função "to_i"
# Para utiliza-lo no range "0..num"
num = gets.chomp.to_i
soma = 0 # Inicializando como um inteiro
for i in 1..num  do # 1..num é um range dos números de 1 a num
  soma += i
end
puts "A soma de 1 a #{num} é #{soma}"

File: python/elem4.py
# coding: utf-8
# int() realiza o typecasting de string para inteiro
num = int(raw_input("Digite um número: "))
soma = 0
# "range(1,num)" é um range dos números de '1' a 'num-1'
for iterador in range(1,num+1):
  soma += iterador

print ("Soma de 1 a %d é %d " % (num,soma))